src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/AuthUser.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import io.micrometer.core.lang.NonNull
```

Comment:
Should we be using the spring import org.springframework.lang.NonNull ?

Reply:
yes - good spot. this is an IDE mishap.